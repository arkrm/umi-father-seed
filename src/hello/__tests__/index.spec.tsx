import React from 'react';
import { render } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Hello from '../index';

describe('Hello Component', () => {
  it('test render', () => {
    const wrapper = render(<Hello />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
