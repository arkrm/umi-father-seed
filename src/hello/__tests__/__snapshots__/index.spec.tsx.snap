// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello Component test render 1`] = `
<div
  class="ufs-hello-wrapper"
>
  <div
    class="ufs-hello-title"
  >
    Hello, umi-seed-father
  </div>
  <div
    class="ufs-hello-content"
  >
    这是 umi-father-seed 示例工程的 Hello 组件
  </div>
</div>
`;
